ONSdigital/sdc-int-rh-ui
# 381 Set up Travis CI

# Motivation and Context

We need to use start using TravisCI to run unit tests on our PRs. 

# What has changed



**Note: Integration tests and Vulture linting are not being checked by TravisCI as they need further work**
* Add Travis build config for running vulnerability check, flake8 and unit tests
* Fix existing Flake8 linting failures
* Remove unused `pyjwt` dependency as this was failing vulnerability check with pipenv 
* Remove redundant cloudbuild config

# How to test?



* Run a `pipenv clean` to remove the redundant package
* Check the commands in Travis work locally

# Links



https://trello.com/c/kPnanCZg
# Screenshots (if appropriate):
